src/pynwb/validate.py

```diff
@@ -45,13 +49,12 @@ def main():
 
     feature_parser = parser.add_mutually_exclusive_group(required=False)
     feature_parser.add_argument("--cached-namespace", dest="cached_namespace", action='store_true',
-                                help="Use the cached namespace (default).")
+            help="Use the cached namespace (default: %(default)s).", default=True)
```

Comment:
```suggestion
            help="Use the cached namespace (default: %(default)).", default=True)
```

Reply:
Did you see https://docs.python.org/3/library/argparse.html?highlight=argumentparser#help? This suggests that the trailing `s` is indeed correct.

The help output also looks correct.

```
  --cached-namespace    Use the cached namespace (default: True).
```

Reply:
My bad! I thought it was a typo and should have checked first.